core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/diagnostics/DiagnosticsCollectionValidator.java

```diff
@@ -48,6 +51,16 @@ public void validate(DiagnosticsDestination destination, Telemetry telemetry, St
         }
     }
 
+    private void validateStackStatus(Stack stack) {
+        Status stackStatus = stack.getStatus();
+        if (Status.STOPPED.equals(stackStatus)
+                || Status.DELETED_ON_PROVIDER_SIDE.equals(stackStatus)
+                || Status.REQUESTED.equals(stackStatus)) {
+            throw new BadRequestException(String.format("Cannot run diagnostics on stack (crn: '%s') if the stack is in %s state",
```

Comment:
I think the crn is not useful for the enduser also do not call it stack but use CDP terms like Data Hub or Data Lake